velox/exec/Operator.h

```diff
@@ -305,6 +305,14 @@ class Operator : public BaseRuntimeStatWriter {
     }
   };
 
+  /// The name of the runtime spill read stats collected and reported by
+  /// operators that supporting spilling.
+  static inline const std::string kSpillReadBytes{"spillReadBytes"};
+  static inline const std::string kSpillCount{"spillReads"};
+  static inline const std::string kSpillReadTimeUs{"spillReadTimeUs"};
+  static inline const std::string kSpillDeserializationTimeUs{
+      "spillDeserializationTimeUs"};
```

Comment:
Why are we specifically storing them as static inline in Operator? Thanks

Reply:
Resolved the comment https://github.com/facebookincubator/velox/pull/9285#discussion_r1545529673.